tests/consumer/src/main/proto/protodata/test/project.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
```

Comment:
I think it would be interesting (and maybe helpful, too) to put this Proto file — or another one, but similar to this one — to `src/test/proto`. And cover the "multiple source sets" feature as well.